apps/root/src/services/transactionService.ts

```diff
@@ -214,22 +226,22 @@ export default class TransactionService extends EventsManager<TransactionService
       } else {
         transactionsHistory.history = {
           ...transactionsHistoryResponse,
+          indexing: Object.entries(transactionsHistoryResponse.indexing).reduce<TransactionsHistory['indexing']>(
+            (acc, [address, indexersData]) => {
+              if (!('error' in indexersData)) {
+                Object.entries(indexersData).forEach(([indexerUnit, indexerData]) => {
+                  // eslint-disable-next-line no-param-reassign
+                  acc[address as Address] = { ...acc[address as Address], [indexerUnit]: indexerData };
+                });
+              }
+              return acc;
+            },
+            {}
+          ),
           events: [...transactionsHistory.history.events, ...transactionsHistoryResponse.events],
         };
       }
 
-      transactionsHistory.history.indexing = Object.entries(transactionsHistoryResponse.indexing).reduce<
-        TransactionsHistory['indexing']
-      >((acc, [address, chainsData]) => {
-        if (!('error' in chainsData)) {
-          Object.entries(chainsData).forEach(([chainId, chainData]) => {
-            // eslint-disable-next-line no-param-reassign
-            acc[address as Address] = { ...acc[address as Address], [Number(chainId)]: chainData };
-          });
-        }
-        return acc;
-      }, {});
-
```

Comment:
Not sure why we moved this into both if-else statements

Reply:
Moved it out :)